### DISCUSSION TITLE
重要委托!

### DISCUSSION THREAD

**Discussion by DJLFi** [share121/inter-knot#3263] [其他]
请立即上火影，拿晓鼬穿三，委托金:1000000金币（找火影官方拿）
![f6bab77bac8916f4b791912e1e13b846](https://github.com/user-attachments/assets/314e625a-173c-43ec-aa20-3aab81b81ce6)

**Comment by gxianqshwo** [comment]
金币真的要的到吗:(

**Comment by Bahajsjsh** [comment]
拿不到是真的拿到了是中幻术🙃

**Comment by nmlg-sb** [comment]
怎么这里也有村里人啊🤓

**Comment by Kevin-Kaslana-Jiemie** [comment]
这就不得不拿出关老湿的自卫鼬了

**Comment by ysdtal** [comment]
无处不在的村里人

**Comment by x18144506** [comment]
立刻上号穿三只晓鼬
